PhysicalTherapy/PhyscialTherapy/Client/src/app/therapist-routine-creation-screen/therapist-routine-creation-screen.component.html

```diff
@@ -20,15 +20,15 @@
     <div [formGroupName]="i">
         <div>
           {{i}}
-          <input formControlName="targetIndex" placeholder="Change positions?" type="number">
-          <input formControlName="exerciseName" placeholder="NAME" list="exerciseList">
+          <input id="target{{i}}" formControlName="targetIndex" placeholder="Change positions?" type="number">
+          <input id="exName{{i}}" formControlName="exerciseName" placeholder="NAME" list="exerciseList">
         </div>
         <div>
-          <input formControlName="holdLength" placeholder="Hold Length">
-          <input formControlName="reps" placeholder="Number of reps">
-          <input formControlName="sets" placeholder="Number of sets">
+          <input id="hold{{i}}" formControlName="holdLength" placeholder="Hold Length">
+          <input id="reps{{i}}" formControlName="reps" placeholder="Number of reps">
+          <input id="sets{{i}}" formControlName="sets" placeholder="Number of sets">
```

Comment:
Why did you add the ID field here instead of accessing these through the form array?

Reply:
Just didn't notice those at first. :P I'll change it.

Reply:
Alright! You can leave them in too. I was just worried that it didn't work as well as I thought. :P

Reply:
I changed it and it still works. :)